src/components/instructions/Instructions.jsx

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from "react";
+import { Button, Container, Row, Col } from "react-bootstrap";
+
+import Howto from "./HowTo";
+
+export default function Instructions(props) {
+  const [showHowto, setShowHowto] = useState(false);
+
+  return (
+    <Container fluid id="information" className="bg-secondary text-light">
+      <Row className="justify-content-md-center py-5">
+        <Col xs={8}>
+          <div className="d-flex flex-column">
+            <h1 className="my-4">Narragansett Bay Volume Renderer</h1>
+            <p className="my-4">
+              The main goal of the project is to provide an easily accessible
```

Comment:
with the renaming of the files, be careful on the merge in of main as this content has changed, but in a different file (so probably some manual moving is required).

Reply:
this looks like the old text - grab the updated text.

Reply:
Ugh it didn't update. The text changes were just in `Instructions` and `Howto`  right?

Reply:
Looks like it was just `Instructions.jsx`